addons/sale/views/sale_menus.xml

```diff
@@ -58,10 +58,6 @@
             <menuitem id="menu_product_template_action"
                 action="product_template_action"
                 sequence="10"/>
-            <menuitem id="menu_products"
-                action="product.product_normal_action_sell"
```

Comment:
Why is this action still present? It should be cleaned up as well.
Check all views and menus.